mars/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for mars project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'mars.settings')
+
+application = get_asgi_application()
```

Comment:
Do have a look at the docs how is this different from `get_wsgi_application` - it's quite new actually - I'm still using `wsgi` with Django 2.2.

Reply:
ASGI is a standard for building async web servers that has been introduced since Django 3.0. However, since Django is written mostly in synchronous style, most of it will have to be "refactored" to make the framework fully async. Currently, with ASGI enabled, the requests are handled in async, while processing is done synchronously.  

I suggest we start using it as it provides [better performance ](https://arunrocks.com/a-guide-to-asgi-in-django-30-and-its-performance/).

Note: ASGI requires a new dependency `asgiref==3.2.3` and all methods for handling requests have to be prefixed with `async`.

Reply:
Nice. I'm excited